popup.js

```diff
@@ -0,0 +1,204 @@
+const worksSection = document.getElementById('work');
+const headerWrapper = document.querySelector('.header');
+
+const works = [
+  {
+    name: 'Tonic',
+    jobDescription: {
+      company: 'CANOPY',
+      role: 'Back End Dev',
+      year: '2015',
+    },
+    description: {
+      featured:
+        'A daily selection of privately personalized reads; no accounts or sign-ups required.',
```

Comment:
- [ optional] you can add the size of your desktop popup a little to make things more vissible.